conda/environments/cudf_dev_cuda10.0.yml

```diff
@@ -12,6 +12,8 @@ dependencies:
   - rmm=0.15.*
   - cmake>=3.14
   - cmake_setuptools>=0.1.3
+  - gtest
+  - gmock
```

Comment:
Do we want to version pin these? @harrism @aucahuasi do you have any opinions?

Reply:
What probably should like we do for other dependencies.

Reply:
These still need to be version pinned

Reply:
Lets just pin to the current latest release on conda-forge and call it a day for now?